src/components/Sidebar/Sidebar.sass

```diff
@@ -5,17 +5,20 @@
   height: auto
   width: 100%
   overflow: scroll
-  position: relative;
+  position: relative
+
+  li
+    font-size: 1.5rem
 
   p
-    font-size: 1.2rem
+    font-size: 1.5rem
     font-style: italic
     color: $dark-gray
     margin-bottom: 1rem
     +bold
-
   .fixed-search
```

Comment:
This was a bandaid fix, since the sidebar was bleeding outside of its container div. I just wanted the UL beneath to remain visible